blockstore/src/main/java/brooklyn/location/blockstore/vclouddirector15/VcloudNewVolumeCustomizer.java

```diff
@@ -0,0 +1,20 @@
+package brooklyn.location.blockstore.vclouddirector15;
+
+import brooklyn.location.blockstore.NewVolumeCustomizer;
+import brooklyn.location.blockstore.api.VolumeManager;
+import brooklyn.location.blockstore.api.VolumeOptions;
+
+import java.util.List;
+
+public class VcloudNewVolumeCustomizer extends NewVolumeCustomizer {
+    private final static VolumeManager volumeManager = new VcloudVolumeManager();
```

Comment:
Better to move this field to `NewVolumeCustomizer`  and instantiate it in constructor.